internal/infrastructure/middleware/rbac_test.go

```diff
@@ -40,8 +40,8 @@ func helperRbacCreateMapping() rbac_data.RBACMap {
 		Add("/domains/:uuid", http.MethodPost, rbac_data.NewRbacPermission(service, resourceDomains, rbac_data.RbacVerbCreate)).
 		Add("/domains/:uuid", http.MethodPatch, rbac_data.NewRbacPermission(service, resourceDomains, rbac_data.RbacVerbUpdate)).
 		Add("/domains/:uuid", http.MethodDelete, rbac_data.NewRbacPermission(service, resourceDomains, rbac_data.RbacVerbDelete)).
-		Add("/signing_keys", http.MethodDelete, rbac_data.NewRbacPermission(service, resourceSigningKeys, rbac_data.RbacVerbExecute)).
-		Add("/host-conf/:inventory_id/:fqdn", http.MethodDelete, rbac_data.NewRbacPermission(service, resourceHostConf, rbac_data.RbacVerbExecute)).
+		Add("/signing_keys", http.MethodPost, rbac_data.NewRbacPermission(service, resourceSigningKeys, rbac_data.RbacVerbExecute)).
```

Comment:
I don't know what these 2 lines mean / what is their impact.  When commented out, the tests still pass.

Reply:
Discussion: these 2 lines should be removed + POST of `domains/:uuid`